deployment/playground/src/search.py

```diff
@@ -87,6 +86,25 @@ def search(attribute_name, attribute_value, jwt, top_k=None, filter_dict=None):
     return call_flow(URL_HOST, data, attribute_name, domain)
 
 
+def get_suggestion(text, jwt):
```

Comment:
Maybe you can omit this function and reuse the search function adding an additional param to it `endpoints` which specifies the host end?
```suggestion
def get_suggestion(text, jwt, endpoint='search'):
.
.
.
URL_HOST = = f"{domain}/api/v1/{params.input_modality}-to-{params.output_modality}/{endpoints}"
```

Reply:
applied